Dockerfile

```diff
@@ -116,7 +116,7 @@ RUN cd eth2-testnet-genesis && PATH="/usr/local/go/bin/:$PATH" go build
 
 # builder and relay
 FROM bitnami/minideb:${DEBIAN_RELEASE} AS ethodbuilder
-ARG ETHDO_REF="v1.31.0"
+ARG ETHDO_REF="5895bbfbe6484505ddf666f0f4c0e25dde3cce9b"
```

Comment:
Why do we need Ethdo?

Reply:
It triggers voluntary exit of validators in the test suite, see https://github.com/ChorusOne/eth-possim/blob/main/tests/test_pos_privatenet.py#L156